plugin-server/src/main/job-queues/job-queue-manager.ts

```diff
@@ -50,7 +58,35 @@ export class JobQueueManager implements JobQueue {
         }
     }
 
-    async enqueue(jobName: string, job: EnqueuedJob): Promise<void> {
+    async enqueue(jobName: string, job: EnqueuedJob, instrumentationContext?: InstrumentationContext): Promise<void> {
+        const jobType = 'type' in job ? job.type : 'buffer'
+        const jobPayload = 'payload' in job ? job.payload : job.eventPayload
+        const pluginServerMode = this.pluginsServer.PLUGIN_SERVER_MODE ?? 'full'
+        await instrument(
+            this.pluginsServer.statsd,
+            {
+                metricName: jobName === JobName.PLUGIN_JOB ? 'vm.enqueuePluginJob' : 'vm.enqueueBufferJob',
```

Comment:
So this is confusing.
- It makes an implicit assertion that there's only two different types of jobs. Which might be true for now, but will it be long-term?
- It prefixes the metric with `vm.` which just is a lie

Suggestion: 

```suggestion
                metricName: `job_queues_enqueue`
```

And add jobName as a tag instead.

Reply:
I was making sure we kept the metric that was already established before, but can get rid of that in favor of this new metric